README.md

```diff
@@ -1,4 +1,4 @@
-# Your GitHub Learning Lab Repository for Introducing GitHub
+### Your GitHub Learning Lab Repository for Introducing GitHub
```

Comment:
@ashahba Thanks for the PR.
Can we try reducing the font size by 50% instead of 75%.

Reply:
@fdarabi I made the changes you requested.
Please review again and let me know.

Thanks.

Reply:
I can't remember where I see the changes you made. like changing the font size. Please advise. i